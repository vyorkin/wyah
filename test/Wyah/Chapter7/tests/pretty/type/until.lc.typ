(TVar (TV "a") :-> TCon "Bool") :-> ((TVar (TV "a") :-> TVar (TV "a")) :-> (TVar (TV "a") :-> TVar (TV "a")))
