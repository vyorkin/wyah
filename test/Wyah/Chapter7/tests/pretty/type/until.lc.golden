(a -> Bool) -> (a -> a) -> a -> a